DSCResources/MSFT_xWebVirtualDirectory/MSFT_xWebVirtualDirectory.psm1

```diff
@@ -9,6 +9,7 @@ function Get-TargetResource
         $Website,
 
         [parameter(Mandatory = $true)]
```

Comment:
So I thought about that but 1) it doesn't accept nulls actually, only empty strings and 2) it's a key parameter so I wasn't sure if it was against best practices to provide a default value. It's up to you, I could see it go either way.